spec/models/user_spec.rb

```diff
@@ -22,7 +22,9 @@
 require 'rails_helper'
 
 RSpec.describe User, type: :model do
-  let!(:user) { create :user }
+  let!(:user) { create :user, opt_out: false }
+  let!(:user2) { create :user, opt_out: false }
+  let!(:user3) { create :user, opt_out: true }
```

Comment:
Please change `user3` to `opted_out_user` so the code reviewer won't have look up here to know what is going on.